src/Spago/Build.hs

```diff
@@ -322,12 +322,26 @@ runBackend
 runBackend maybeBackend RunDirectories{ sourceDir, executeDir } moduleName maybeSuccessMessage failureMessage extraArgs = do
   logDebug $ display $ "Running with backend: " <> fromMaybe "nodejs" maybeBackend
   BuildOptions{ pursArgs } <- view (the @BuildOptions)
-  let postBuild = maybe (nodeAction $ Path.getOutputPath pursArgs) backendAction maybeBackend
+  isES <- Purs.hasMinPursVersion "0.15.0"
+  let postBuild = maybe (nodeAction isES $ Path.getOutputPath pursArgs) backendAction maybeBackend
   build (Just postBuild)
   where
     fromFilePath = Text.pack . Turtle.encodeString
     nodeArgs = Text.intercalate " " $ map unBackendArg extraArgs
-    nodeContents outputPath' =
+    esContents outputPath' =
+      fold
+        [ "import { main } from '"
+        , Text.replace "\\" "/" (fromFilePath sourceDir)
+        , "/"
+        , Text.pack outputPath'
+        , "/"
+        , unModuleName moduleName
+        , "/"
+        , "index.js"
+        , "'\n\n"
+        , "main()"
```

Comment:
Should there be semi-colons after `index.js` and `main()`?

Reply:
They are not required. Personally I dislike them as they don't add anything except noise

Reply:
Ah, ok. I'll mark this as resolved.